CMakeLists.txt

```
@@ -59,5 +66,6 @@ install(EXPORT ${PROJECT_NAME}Targets
   DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}/"
   )
 
-gaudi_install(CMAKE cmake/${PROJECT_NAME}Config.cmake)
+include(cmake/k4FWCoreCreateConfig.cmake)
 
+gaudi_install(CMAKE ${PROJECT_BINARY_DIR}/k4FWCoreConfig.cmake)
```

```suggestion
install(FILES ${PROJECT_BINARY_DIR}/k4FWCoreConfig.cmake DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/k4FWCore)
```
It's a single cmake file, `gaudi_install` doesn't do anything special for us and this is clearer (and we can check that the paths here and given to `configure_package_config_file` match).

---

Done.